docs/source/release_notes.rst

```diff
@@ -3,6 +3,51 @@
 =================
 
 
+v0.8.0 (2020-05-18)
+===================
+
+Description
+-----------
+
+A major new feature release with added views for complex devices and
+simplified configurability.
+
+Enhancements / What's New
+-------------------------
+
+-  Panels: New ``TyphosCompositeSignalPanel``, which composes multiple
+   ``TyphosDisplay``\ s in a tree-like view.
+-  Benchmarking: new profiling tools accessible in the command-line
+   ``typhos`` tool, allowing for per-line profiling of standardized
+   devices. (``--benchmark``)
+-  Template discovery: templates are discovered based on screen macros
+   and class inheritance structure, with the fallback of built-in
+   templates.
+-  New command-line options for testing with mock devices
+   (``--fake-device``).
+-  Performance: Major performance improvements by way of background
+   threading of signal description determination, display path caching,
+   and connection status monitoring to reduce GUI thread blocking.
+-  Display: Adds a "display switcher" tool for easy access to different
+   screen types.
+-  Display: Adds a "configuration" button to displays.
+-  Filtering: Filter panel contents by kinds.
+-  Filtering: Filter panel contents by signal names.
+-  Setpoint history: a history of previous setpoints has been added to
+   the context menu in ``TyphosLineEdit``.
+-  Positioners have been reworked to be less magical.
+
+Compatibility / fixes
+---------------------
+
+-  The deprecated name ``Typhon`` has been removed, as planned.
```

Comment:
Surely the full removal of `typhon` as a name warrants a major version increment? Unless we feel like there's something special about a `1.0.0` release and we'd rather hold off.

Reply:
At the very least it deserves a special call-out as an API change

Reply:
OK, will adjust version numbers.